src/ratelimitedfunc.js

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright 2016 OpenMarket Ltd
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+module.exports = function(f, min_interval_ms) {
+    this.lastCall = 0;
+    this.scheduledCall = undefined;
```

Comment:
`camelCase` or `under_scores`, which is it? We don't really fix on a certain style in react/vector, but we should at least be consistent within a file (especially new files).